interchain-security/dungeon-1/proposal-dungeon.json

```
@@ -0,0 +1,18 @@
+{
+    "title": "Create the dungeon chain",
+    "description": "This is the proposal to create the \"dungeon\" consumer chain.**",
+    "chain_id": "dungeon-1",
+    "initial_height": {
+        "revision_height": 1
+    },
```

If you want to use a `-1` suffix in your chain ID, you will need to add a `revision_number` key:
```suggestion
    "initial_height": {
        "revision_height": 1,
        "revision_number: 1
    },
```

---

5e7cb4f3e42f9a57bcc51c2395a39fc019f13a50